pkgs/by-name/li/librandombytes/package.nix

```diff
@@ -0,0 +1,81 @@
+{
+  stdenv,
+  lib,
+  python3,
+  openssl,
+  fetchzip,
+}:
+stdenv.mkDerivation (prev: {
```

Comment:
Oh, also: I'd prefer to replace `prev` by `finalAttrs` here, as there are 2200+ occurrences of `stdenv.mkDerivation (finalAttrs:` in Nixpkgs and 0 for this.